go.mod

```diff
@@ -4,5 +4,7 @@ go 1.14
 
 require (
 	github.com/aws/aws-sdk-go v1.31.5
+	github.com/mitchellh/go-homedir v1.1.0
```

Comment:
can you add these new pkgs to the third party licenses file

Reply:
good catch!